Missing header in dependency

The header seems to not exist any more:
```
make
[ 28%] Built target envire_urdf
[ 57%] Built target envire_urdf_bin
[ 71%] Building CXX object test/CMakeFiles/test_suite.dir/test_robot.cpp.o
/home/dfki.uni-bremen.de/afabisch/tmp/envire/envire/envire_urdf/test/test_robot.cpp:4:42: fatal error: envire_core/graph/GraphViz.hpp: No such file or directory
compilation terminated.
test/CMakeFiles/test_suite.dir/build.make:86: recipe for target 'test/CMakeFiles/test_suite.dir/test_robot.cpp.o' failed
make[2]: *** [test/CMakeFiles/test_suite.dir/test_robot.cpp.o] Error 1
CMakeFiles/Makefile2:179: recipe for target 'test/CMakeFiles/test_suite.dir/all' failed
make[1]: *** [test/CMakeFiles/test_suite.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/test/test_robot.cpp
+++ b/test/test_robot.cpp
@@ -1,7 +1,7 @@
 #include <boost/test/unit_test.hpp>
 #include <envire_urdf/GraphLoader.hpp>
 #include <envire_core/graph/EnvireGraph.hpp>
-#include <envire_core/graph/GraphViz.hpp>
+#include <envire_core/graph/GraphDrawing.hpp>
 #include <envire_core/items/Item.hpp>
 
 const std::string path="../../test/urdf/two_boxes.urdf";
@@ -19,7 +19,7 @@ BOOST_AUTO_TEST_CASE(initRobotGraph_noPos)
 {
     std::shared_ptr<envire::core::EnvireGraph>  transformGraph( new envire::core::EnvireGraph());
     urdf::ModelInterface robot = *urdf::parseURDFFile(path).get();
-    envire::core::GraphViz viz;
+    envire::core::GraphDrawing viz;
     envire::urdf::GraphLoader loader(transformGraph);
     loader.loadStructure(robot);
     viz.write(*transformGraph, "initRobotGraph_noPos_Test.dot");
@@ -29,7 +29,7 @@ BOOST_AUTO_TEST_CASE(initRobotGraph_withPos)
 {
     std::shared_ptr<envire::core::EnvireGraph>  transformGraph( new envire::core::EnvireGraph());
     urdf::ModelInterface robot = *urdf::parseURDFFile(path).get();
-    envire::core::GraphViz viz;
+    envire::core::GraphDrawing viz;
     std::cout << "Path to robot model " << path << std::endl;
     envire::core::Transform iniPose;
     iniPose.transform.orientation = base::Quaterniond::Identity();
@@ -45,7 +45,7 @@ BOOST_AUTO_TEST_CASE(initRobotGraph_withDynamicJoint)
 {   
     std::shared_ptr<envire::core::EnvireGraph>  transformGraph( new envire::core::EnvireGraph());
     urdf::ModelInterface robot = *urdf::parseURDFFile(pathDynamic).get();
-    envire::core::GraphViz viz;
+    envire::core::GraphDrawing viz;
     std::cout << "Path to robot model " << pathDynamic << std::endl;
     envire::core::Transform iniPose;
     iniPose.transform.orientation = base::Quaterniond::Identity();
@@ -74,7 +74,7 @@ BOOST_AUTO_TEST_CASE(loadFrames)
 {
     
     urdf::ModelInterface robot = *urdf::parseURDFFile(pathDynamic);
-    envire::core::GraphViz viz;
+    envire::core::GraphDrawing viz;
     envire::urdf::GraphLoader loader = getRobotWithInitGraph(robot);
     loader.loadFrames(robot);
     viz.write(*loader.getGraph(), "loadFrames_Test.dot");
@@ -83,7 +83,7 @@ BOOST_AUTO_TEST_CASE(loadFrames)
 BOOST_AUTO_TEST_CASE(loadFrames_withDynamicJoint)
 {
     urdf::ModelInterface robot = *urdf::parseURDFFile(pathDynamic);
-    envire::core::GraphViz viz;
+    envire::core::GraphDrawing viz;
     envire::urdf::GraphLoader loader = getRobotWithInitGraph(robot);
     loader.loadFrames(robot);
     viz.write(*loader.getGraph(), "loadFrames_withDynamicJoint_Test.dot");
@@ -92,7 +92,7 @@ BOOST_AUTO_TEST_CASE(loadFrames_withDynamicJoint)
 BOOST_AUTO_TEST_CASE(loadJoints)
 {
     urdf::ModelInterface robot = *urdf::parseURDFFile(pathDynamic);
-    envire::core::GraphViz viz;
+    envire::core::GraphDrawing viz;
     envire::urdf::GraphLoader loader = getRobotWithInitGraph(robot);
     loader.loadJoints(robot);
     viz.write(*loader.getGraph(), "loadJoints_Test.dot");
@@ -101,7 +101,7 @@ BOOST_AUTO_TEST_CASE(loadJoints)
 BOOST_AUTO_TEST_CASE(completeTest)
 {
     urdf::ModelInterface robot = *urdf::parseURDFFile(pathDynamic);
-    envire::core::GraphViz viz;
+    envire::core::GraphDrawing viz;
     envire::urdf::GraphLoader loader = getRobotWithInitGraph(robot);
     loader.loadFrames(robot);
     loader.loadJoints(robot);

